Makefile

```diff
@@ -114,3 +114,8 @@ pack:
 .PHONY: test-env
 test-env:
 	docker run --privileged -v /sys/fs/cgroup:/sys/fs/cgroup -v $(PWD):$(PWD) -w $(PWD)  --rm -it quay.io/footloose/debian10:0.6.3 /lib/systemd/systemd
+
+.PHONY: lint
+lint:
+	which golangci-lint  2>&1 > /dev/null || curl -sSfL https://raw.githubusercontent.com/golangci/golangci-lint/master/install.sh | sh -s -- -b $(go env GOPATH)/bin
```

Comment:
Instead of which we could use `command` to identify is binary is available or not, please see: [here](https://stackoverflow.com/questions/592620/how-can-i-check-if-a-program-exists-from-a-bash-script)

Reply:
The pattern of "which util || install util"  is used throughout the make file.  Should I change it everywhere or would it be better do do so in a separate patch?

Reply:
IMO it should be changes everywhere and can be a part of this patch, it is only replacing the command. But can be a separate patch as well to keep a clean commit history